lib/registrant.rb

```diff
@@ -0,0 +1,24 @@
+class Registrant
+
+    attr_reader :name, :age, :license_data
+
+    def initialize(name, age, permit=false)
+        @name = name
+        @age = age
+        @permit = permit
+        @license_data = {:written=>false, :license=>false, :renewed=>false}
+    end
+
+    def permit?
+        @permit
+    end
+
+    def earn_permit 
+        if @age >= 16
```

Comment:
Technically there wasn't an age restriction on early a permit, only on earning a licesnse. If there was an age restriction, this would be perfect! :)